coordinator/src/routes.rs

```diff
@@ -401,3 +402,9 @@ pub async fn get_metrics(State(state): State<Arc<AppState>>) -> impl IntoRespons
 
     (StatusCode::OK, open_telemetry_metrics + &autometrics)
 }
+
+pub async fn get_health() -> Result<Json<String>, AppError> {
+    // TODO: Implement any health check logic we'd need
+    // So far this just returns if the server is running
+    Ok(Json("Server is healthy".to_string()))
```

Comment:
🙃 Maybe `{ is_healthy: true }` rather than a string?

Reply:
Given the current implementation of a simple health endpoint that can be either queried or not (because the server is not available) I would expect an empty `200` as response. 

Eventually we could make a more elaborate endpoint that could return an object that could provide a reason as well (e.g. the oracle is not reachable in the coordinator).

Reply:
The idea is to return an enum with a reason, but it's not in scope of this ticket.

I prefer keeping a string rather than nothing as it's more readable when you manually query it until we have a nicer response there.